### PR TITLE
MyProfile

### PR DESCRIPTION
**Pull request by Noibisjunior**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2098"></a>

### CODE REVIEW

File: public/data/Noibisjunior.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      "color": "green",
+      "description": "Started my software development bootcamp with freeCodeCamp",
+      "url": "https://github.com/Noibisjunior"
+    },
+    {
+      "title": "Started FreeLancing as a FrontEnd Developer",
+      "date": "September 2022",
+      "icon": "code",
+      "color": "blue",
+      "url": "Noibisjunior22@gmail.com"
```
Comment (kumarsonsoff3) [inline public/data/Noibisjunior.json:32]: It should be `mailto:Noibisjunior22@gmail.com`. Please update it with the correct syntax for the mail url! 
```suggestion
      "url": "mailto:Noibisjunior22@gmail.com"
```

Comment (kumarsonsoff3) [reply public/data/Noibisjunior.json:32]: You could also commit these changes directly by using the `Commit suggestion` button above!

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by Rohith-JN — APPROVED** [review]

File: public/data/Noibisjunior.json (hunk 1 above)
Comment (eddiejaoude) [reply public/data/Noibisjunior.json:32]: great idea 👍

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Pull request merged**